README.md

```diff
@@ -157,6 +178,29 @@ Typically Vagrant box deployments encounter one of few problems:
 
     uaa is not functioning, try steps above
 
+* volumes don't get mounted when suspending/resuming the box
+
+  For now only `vagrant stop` and then `vagrant up` fixes it.
+
+* When restarting the box with either `vagrant reload` or `vagrant stop/up` some
+  pods never come up automatically. You have to do a `make stop` and then
+  `make run` to bring this up.
+
+* Pulling images during any of `vagrant up` or `make vagrant-prep` or `make docker-deps`
+  fails.
+
+  Make sure you have enabled ipv4 ip fowarding.
+
+  Temporarily:
+
+  ```echo "1" | sudo tee /proc/sys/net/ipv4/ip_forward```
```

Comment:
According to the [docs](https://docs.docker.com/engine/userguide/networking/default_network/container-communication/#communicating-to-the-outside-world) it should be the default. I don't know why it isn't in our case but I think we don't only need to enable this inside the vagrant box but also in the host system. If I recall correctly I didn't have access to the internet from inside the vagrant box when the host's value was not set to "1".